lib/plugins/invoke/invoke.js

```diff
@@ -37,6 +37,29 @@ class Invoke {
             shortcut: 'l',
           },
         },
+        commands: {
+          local: {
+            usage: 'Invoke lambda locally',
```

Comment:
"Invoke function locally" because other providers might not call the function "lambda"